bin/offline.cpp

```diff
@@ -101,6 +101,7 @@ int main(int argc, char *argv[]) {
     args::ValueFlag<double> minZoomValue(argumentParser, "number", "Min zoom level", {"minZoom"});
     args::ValueFlag<double> maxZoomValue(argumentParser, "number", "Max zoom level", {"maxZoom"});
     args::ValueFlag<double> pixelRatioValue(argumentParser, "number", "Pixel ratio", {"pixelRatio"});
+    args::ValueFlag<bool> includeIdeographsValue(argumentParser, "boolean", "Include CJK glyphs", {"includeIdeographs"});
```

Comment:
Consider changing this so that the flag is `--excludeIdeographs` if the default behavior is to include them.

Reply:
On the SDK side, we decided to go with default include because it preserves existing behavior. Do you think we can get away with a breaking change on the command-line side? I don't have a very strong understanding of how this tool is typically used, but I think defaulting to _not_ include ideographs is usually better.

Reply:
Going with default behavior of not including them.